.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '-vv --color=yes'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.12'
```

Comment:
Maybe it's too far, but should we extract this into a matrix? No idea if different Python versions are as painful as Ruby ones though.